mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -13,36 +16,48 @@ CLASS ail net/minecraft/entity/Entity
 	FIELD O age I
 	FIELD P insideWater Z
 	FIELD Q waterHeight D
+	FIELD R inWater Z
 	FIELD S inLava Z
+	FIELD T timeUntilRegen I
+	FIELD U firstUpdate Z
 	FIELD V dataTracker Lql;
 	FIELD W FLAGS Lqi;
 	FIELD X POSE Lqi;
+	FIELD Y updateNeeded Z
 	FIELD Z chunkX I
 	FIELD aA NAME_VISIBLE Lqi;
 	FIELD aB SILENT Lqi;
 	FIELD aC NO_GRAVITY Lqi;
 	FIELD aD invulnerable Z
 	FIELD aE scoreboardTags Ljava/util/Set;
+	FIELD aF teleportRequested Z
 	FIELD aG pistonMovementDelta [D
 	FIELD aH pistonMovementTick J
 	FIELD aI size Laim;
 	FIELD aJ standingEyeHeight F
 	FIELD aa chunkY I
 	FIELD ab chunkZ I
+	FIELD ac trackedX J
+	FIELD ad trackedY J
+	FIELD ae trackedZ J
 	FIELD af ignoreCameraFrustum Z
 	FIELD ag velocityDirty Z
 	FIELD ah portalCooldown I
 	FIELD ai inPortal Z
 	FIELD aj portalTime I
 	FIELD ak dimension Lbyh;
 	FIELD al lastPortalPosition Lev;
+	FIELD am lastPortalDirectionVector Lcsa;
+	FIELD an lastPortalDirection Lfa;
 	FIELD ao uuid Ljava/util/UUID;
 	FIELD ap uuidString Ljava/lang/String;
 	FIELD aq glowing Z
 	FIELD ar passengerList Ljava/util/List;
 	FIELD as vehicle Lail;
 	FIELD at velocity Lcsa;
-	FIELD au boundingBox Lcrv;
+	FIELD au entityBounds Lcrv;
+	FIELD av nextStepDistance F
+	FIELD aw aerialStepDelta F
```

Comment:
This isn't a delta (delta means difference), it's the value of `distanceWalked` at which the next step sound should be played:

```java
    protected float calculateStepDelta() {
        return (float) ((int) this.distanceWalked + 1);
    }
```